src/components/ComponentPages/TopicDetails/index copy.tsx

```diff
@@ -0,0 +1,250 @@
+import { Typography } from "antd";
```

Comment:
Is this file required ? I see `index copy.tsx` in the filename

Reply:
removed